types/vcards-js/index.d.ts

```diff
@@ -0,0 +1,247 @@
+// Type definitions for vcards-js 2.1
+// Project: https://github.com/enesser/vCards-JS
+// Definitions by: Ben Allfree <https://github.com/benallfree>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface vCard {
```

Comment:
What is probably missing in the interface is the `anniversary` property:
https://github.com/enesser/vCards-js/commit/bf1cee0e51244731fe1e86fb841a4e9571efd78e
https://github.com/enesser/vCards-js/blob/99a298c72d3cf31741eaa39e9c66df973e9fec91/lib/vCardFormatter.js#L166
https://github.com/enesser/vCards-js/blob/master/README.md#L139

Reply:
also `isOrganization`
https://github.com/enesser/vCards-js/blob/master/README.md#L231
https://github.com/enesser/vCards-js/blob/99a298c72d3cf31741eaa39e9c66df973e9fec91/lib/vCardFormatter.js#L389